astropy/coordinates/sky_coordinate.py

```diff
@@ -172,6 +173,15 @@ def __init__(self, *args, **kwargs):
             raise ValueError('Cannot create a SkyCoord without data')
 
     @property
+    def info(self):
+        """
+        Mixin column information (attributes)
+        """
+        if not hasattr(self, '_info'):
+            self._info = ColumnInfo()
+        return self._info
```

Comment:
OK, I need to think better. Though we do have a different subclass in `Quantity`, but that could just be defined like you did above.